qml/Ping360ControlPanel.qml

```diff
@@ -68,7 +68,7 @@ Item {
                 value: ping.transmit_frequency
                 control.from: 500
                 control.to: 1000
-                control.stepSize: 20
+                control.stepSize: 10
```

Comment:
Why not steps of 1 ? Since we merged the wheel control this will makes the user input more precise. There's some users that really want to use precise numbers.

Reply:
Ok, will do